### PR TITLE
Added Tube Geometry example

### PR DESCRIPTION
**Pull request by Temdog007**
tube geometry demo

### CODE REVIEW

**Comment by Mugen87** [comment]
For testing: https://raw.githack.com/Temdog007/three.js/example/TubeGeometryExample/examples/webgl_geometry_tube.html

File: examples/webgl_geometry_tube.html [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+                gui.add( parameters, 'closed' ).name( 'Closed' ).onChange( updateObject );
+
+                gui.add( parameters, 'divisions', 2, 16, 1 ).name( "Divisions" ).onChange( setPoints );
+                gui.add( parameters, 'opacity', 0, 1, 0.01 ).name( "Opacity" ).onChange( updateOpacity );
+
+                gui.add( parameters, 'curve', [ 'CatmullRomCurve3', 'CubicBezierCurve3', 'LineCurve3', 'QuadraticBezierCurve3' ] ).name( 'Curve' ).onChange( updateObject );
+                gui.add( parameters, 'side', [ 'FrontSide', 'BackSide', 'DoubleSide' ] ).name( 'Side' ).onChange( updateSide );
+
+
+                stats = new Stats();
```
Comment (Mugen87) [inline examples/webgl_geometry_tube.html]: `stats` is never defined.

**Comment by Mugen87** [comment]
There is already: have https://threejs.org/docs/index.html#api/en/geometries/TubeGeometry

TBH, I don't see a great benefit of adding an additional example.

File: examples/webgl_geometry_tube.html [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+                var gui = new dat.GUI( { width: 300 } );
+                gui.add( parameters, 'tubularSegments', 3, 128, 1 ).name( 'Tubular Segments' ).onChange( updateObject );
+                gui.add( parameters, 'radius', 0.1, 16, 0.01 ).name( 'Radius' ).onChange( updateObject );
+                gui.add( parameters, 'radialSegments', 3, 16, 1 ).name( 'Radial Segments' ).onChange( updateObject );
+                gui.add( parameters, 'closed' ).name( 'Closed' ).onChange( updateObject );
+
+                gui.add( parameters, 'divisions', 2, 16, 1 ).name( "Divisions" ).onChange( setPoints );
+                gui.add( parameters, 'opacity', 0, 1, 0.01 ).name( "Opacity" ).onChange( updateOpacity );
+
+                gui.add( parameters, 'curve', [ 'CatmullRomCurve3', 'CubicBezierCurve3', 'LineCurve3', 'QuadraticBezierCurve3' ] ).name( 'Curve' ).onChange( updateObject );
```
Comment (Mugen87) [inline examples/webgl_geometry_tube.html]: If I select `LineCurve3` and then `CatmullRomCurve3` again, I only see two control points and not the original ones. Is this intended?

File: examples/webgl_geometry_tube.html [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				font-family: Monospace;
+				background-color: #222;
+				margin: 0px;
+				overflow: hidden;
+			}
+            #info {
```
Comment (Mugen87) [inline examples/webgl_geometry_tube.html]: The file contains formatting issues. It seems you are mixing hard and soft tabs.

File: examples/webgl_geometry_tube.html (hunk 2 above)
Comment (Temdog007) [reply examples/webgl_geometry_tube.html]: I wasn't aware there was already an example for Tube Geometry. I only checked examples page. So now that this example doesn't have a purpose, I'll just close this pull request.

**Pull request closed without merging**